crates/aptos/src/move_tool/mod.rs

```diff
@@ -83,6 +83,8 @@ pub enum MoveTool {
     Prove(ProvePackage),
     Document(DocumentPackage),
     TransactionalTest(TransactionalTestOpts),
+    CreateResourceAccountAddress(CreateResourceAccountAddress),
```

Comment:
What's the use case for creating a resource account from off-chain outside of a standalone module?

Reply:
just consolidated those two commands into one!